### DISCUSSION TITLE
Does the scope capability in JWT works

### DISCUSSION THREAD

**Discussion by letzya** [TykTechnologies/tyk-operator#289] [Q&A] [answered]
I see the [reference](https://github.com/TykTechnologies/tyk-operator/blob/8e62cb7797ce33f4a672f581ee3c863657257015/helm/crds/crds.yaml#L523) to it but it's not in the example files, so just wanted to confirm before trying.
Thanks

**Comment by gernest** [comment, marked as answer]
Yes it works.

If you don't mind after trying can you share the sample so we can add it here and maybe have a BDD tests to cover this.
